tests/views/test_response_correlation.py

```diff
@@ -131,3 +131,49 @@ def test_show_respo_with_obs(mock_data, dash_duo):
     obs_radio_btn.click()
     obs_responses = get_options(dash_duo, "#" + resp_selector_id)
     assert obs_responses == expected_obs_responses
+
+
+def test_response_overview_plot_survives_reload(
+    mock_data,
+    dash_duo,
+):
+    plugin = setup_plugin(dash_duo, __name__, ResponseCorrelation)
+
+    select_ensemble(dash_duo, plugin)
+    select_by_name(
+        dash_duo,
+        f"#{plugin.uuid('parameter-selector-multi-resp')}",
+        "SNAKE_OIL_GPR_DIFF",
+    )
+    select_by_name(
+        dash_duo,
+        f"#{plugin.uuid('parameter-selector-multi-param')}",
+        "BPR_138_PERSISTENCE",
+    )
+
+    container_id = f"#{plugin.uuid('response-overview')}"
+
+    # check that plot is OK
+    check_response_overview_plot_is_drawn(dash_duo, container_id)
+
+    # reload page
+    dash_duo.driver.refresh()
+
+    # check that plot is still OK
```

Comment:
I think you need to wait a bit for all the graphs to be redrawn after the refresh before checking anything:

adding 
```
    # wait a bit for the graph to be drawn
    wait_a_bit(dash_duo, 1)
```
above this line should do the trick and avoid the `stale element error`

Reply:
after a bit more discussion, we decide to scrap the test as removing the flakiness as suggested here makes the test moot :cow:.